scripts/typos_rollout.py

```
@@ -345,38 +594,80 @@ def _write_remote_cache(
+    age_seconds = max(0, min(int(time.time() - cache.stat().st_mtime), 999_999_999))
+    LOGGER.warning(
+        "dictionary refresh degraded operation=https-refresh category=%s "
+        "status=%s cache_age_seconds=%d",
+        category,
+        status if status is not None else "none",
+        age_seconds,
+    )
+    return RefreshResult("stale-cache", cache)
+
+
 def _refresh_http(
```

Ruff (0.16.0)

[warning] Too many return statements (7 > 6)

(PLR0911)